marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/AbstractSparkConnectorTest.java

```diff
@@ -166,4 +171,13 @@ protected GenericInternalRow buildRow(String... values) {
         }
         return new GenericInternalRow(rowValues);
     }
+
+    protected ObjectNode getDatabaseProperties(String database) {
```

Comment:
This method seems to be duplicated in HubTestbase

Reply:
Let's test this via a test in the core project instead, as there's nothing specific to the connector here. In fact, as @srinathgit points out, you can likely just add a new test method to TemporalWriteTest that hits the bulkIngester endpoint.

Reply:
Moved the test to TemporalWriteTest